src/lib.rs

```diff
@@ -117,16 +122,35 @@ pub fn generate_flamegraph_by_running_command<
     workload: String,
     flamegraph_filename: P,
 ) {
+    // handle SIGINT by doing nothing
```

Comment:
I'll let @spacejam comment on this, but personally I think I'd like to see a comment here explaining why we install an empty signal handler. Just to aid the curious reader who happens to come across this code a long time for now :)

Reply:
Indeed, something more useful would help. What about this bit based on the commit message?

"Handle SIGINT with an empty handler, causing only the tracing child to terminate upon delivery. This way, users can interrupt execution and still get a flamegraph from the sample data collected so far." 

Thinking about it, it could also make sense to deregister the handler as soon as the child has terminated to do the right thing when a sigint is delivered later on.

Reply:
Yeah, I like that more detailed comment!